core/src/utils/prove.rs

```diff
@@ -356,11 +356,11 @@ where
     OpeningProof<SC>: Send + Sync,
     ShardMainData<SC>: Serialize + DeserializeOwned,
 {
-    #[cfg(debug_assertions)]
-    {
-        let record_clone = record.clone();
-        machine.debug_constraints(pk, record_clone);
-    }
+    //#[cfg(debug_assertions)]
```

Comment:
Nit: remove before merge.
FYI, we have a cargo-profile called dev-ci, which will be as efficient to run as `--release`, but has debug assertions!